src/main/java/dev/jlibra/KeyUtils.java

```diff
@@ -13,11 +13,16 @@
 
 public class KeyUtils {
 
-    public static String toLibraAddress(byte[] publicKeyBytes) {
+    public static String toHexStringLibraAddress(byte[] publicKeyBytes) {
         SHA3.DigestSHA3 digestSHA3 = new SHA3.Digest256();
         return new String(Hex.encode(digestSHA3.digest(stripPublicKeyPrefix(publicKeyBytes))));
```

Comment:
You could rewrite this now as `return new String(Hex.encode(toByteArrayLibraAddress(publicKeyBytes)));`

Reply:
You're right, fixed.